packages/apollo-federation/package.json

```diff
@@ -20,6 +20,6 @@
     "lodash.xorby": "^4.7.0"
   },
   "peerDependencies": {
-    "graphql": "^14.0.2"
+    "graphql": "^14.6.0"
```

Comment:
@abernix Can you update this peer dependency to `14.6.0`?
It shouldn't be any breaking changes between `14.0.2` and `14.6.0` so it should be a pretty safe update for end-users.
But if it is not possible we can try to figure out other ways to support both `^14.0.2` and `15.0.0` in your code base.